packages/slice-machine/components/Changelog/versionDetails/index.tsx

```diff
@@ -51,13 +51,33 @@ export const VersionDetails: React.FC<VersionDetailsProps> = ({
           <VersionKindLabel versionKind={selectedVersion.kind} />
         )}
       </Flex>
+
       <Flex
         sx={{
           flexDirection: "column",
-          padding: "24px 32px",
+          margin: "24px 32px",
           gap: "24px",
         }}
       >
+        {selectedVersion?.releaseNote?.includes("# Breaking Change") && (
```

Comment:
No need of selectedVersion? as this component take `PackageVersion`

Reply:
yes indeed :')